inc/storefront-template-functions.php

```diff
@@ -127,7 +127,20 @@ function storefront_credit() {
 		<div class="site-info">
 			<?php echo esc_html( apply_filters( 'storefront_copyright_text', $content = '&copy; ' . get_bloginfo( 'name' ) . ' ' . date( 'Y' ) ) ); ?>
 			<?php if ( apply_filters( 'storefront_credit_link', true ) ) { ?>
-			<br /> <?php printf( esc_attr__( '%1$s designed by %2$s.', 'storefront' ), 'Storefront', '<a href="http://www.woocommerce.com" title="WooCommerce - The Best eCommerce Platform for WordPress" rel="author">WooCommerce</a>' ); ?>
+			<br /> <?php printf(
+					wp_kses(
+						__( 'Built with <a href="%1$s" target="_blank" title="WooCommerce - The Best eCommerce Platform for WordPress" rel="author">Storefront & WooCommerce</a>.', 'storefront' ),
```

Comment:
I'd remove the HTML entirely from this string - use placeholders for the opening and closing tag. This means translators won't bugger it up, and it means we don't need to translate our SEO title tag thing in there. It also means we can use esc_html and not rely on kses :)